packages/flutter_localizations/lib/src/l10n/material_en.arb

```diff
@@ -154,4 +154,9 @@
   "@timePickerMinuteModeAnnouncement": {
     "description": "The audio announcement made when the time picker dialog is set to minute mode."
   }
+
+  "modalBarrierDismissLabel": "Dismiss",
+  "@modalBarrierDismissLabel": {
+    "description": "Accessibility label used to annotate a dismissible modal barrier."
```

Comment:
This description will probably not help translators. You could include some of the text you wrote for routes.dart.